mkt/account/tests/test_views.py

```diff
@@ -326,7 +326,7 @@ def test_installed_pagination(self):
         eq_(data['meta']['limit'], 2)
         prev = urlparse(data['meta']['previous'])
         eq_(next.path, self.list_url)
-        eq_(QueryDict(prev.query).dict(), {u'limit': u'2', u'offset': u'0'})
```

Comment:
This is just a change in next/prev url generation, the 'offset' parameter isn't included when it's 0 since that is the default.